File: implementation/scipamato/scipamato-public-web/src/test/java/ch/difty/scipamato/publ/config/DbPropertiesTest.java
package ch.difty.scipamato.publ.config;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;

public class DbPropertiesTest {

    private final DbProperties dbProperties = new DbProperties();

    @Test
    public void schema_hasDefaultValuePublic() {
        assertThat(dbProperties.getSchema()).isEqualTo("public");
    }
}

File: core/core-web/src/test/kotlin/ch/difty/scipamato/core/config/DbPropertiesTest.kt
package ch.difty.scipamato.core.config

import org.amshove.kluent.shouldBeEqualTo
import org.junit.jupiter.api.Test

internal class DbPropertiesTest {

    private val dbProperties = DbProperties()

    @Test
    fun schema_hasDefaultValuePublic() {
        dbProperties.schema shouldBeEqualTo "public"
    }
}
